docs.html

```diff
@@ -0,0 +1,1806 @@
+                <li class="nav-link-item"><a href="#Form">Form</a></li>
+                <li class="nav-link-item"><a href="#Image">Image</a></li>
+                <li class="nav-link-item"><a href="#Lists">Lists</a></li>
+                <li class="nav-link-item"><a href="#Modal">Modal</a></li>
+                <li class="nav-link-item"><a href="#Navigation">Navigation</a></li>
+                <li class="nav-link-item"><a href="#Rating">Rating</a></li>
+                <li class="nav-link-item"><a href="#Slider">Slider</a></li>
+                <li class="nav-link-item"><a href="#Grid">Grid Simplified</a></li>
+                <li class="nav-link-item"><a href="#Typography">Typography</a></li>
+                <li class="nav-link-item"><a href="#Toast">Toast</a></li>
+            </ul>
+            <div style="height: 1.5rem;">
+
+            </div>
+        </div>
+
+        <section class="main grid-item">
+
+            <div id="Alerts">
+                
+                <div class="component-intro">
+                    <h1>Alerts</h1>
+                    <p class="body-cp-lg text-light-weight">Alerts are used to attract user's attention for importatnt information without interrupting the user's flow.</p>
+                </div>
+                <h2 class="h4">Example of Alert Bar without close button</h2>
+                <p>Alerts are available in 5 types- success alert, error alert, warning alert, primary alert, secondary alert.<br> To use this alert just use class name <span class="primary-text-color">alert-box</span> and also add class according to alert type- <span class="primary-text-color">alert-primary, alert-secondary, alert-danger, alert-success, alert-warning</span>. (e.g.<span class="primary-text-color">class="alert-box alert-primary"</span>).<br>You can copy html part from below code snippet.</p>
+                <div class="component-display-container display-flex-column">
+                    <div class="alert-box alert-primary">A primary alert example with a text link <a class="link-text link-text-primary">Check it out!</a>
+                    </div>
+                    <div class="alert-box alert-secondary">A secondary alert example!!!!
+                    </div>
+                    <div class="alert-box alert-success"><span class="alert-icon"><i class="fas fa-check-circle"></i></span>A success alert example!!!!
+                    </div>
+                    <div class="alert-box alert-danger"><span class="alert-icon"><i class="fas fa-exclamation-triangle"></i>
+                    </span>A danger alert example!!!!
+                    </div>
+                    <div class="alert-box alert-warning"><span class="alert-icon"><i class="fas fa-exclamation-triangle"></i></span> A warning alert example!!!
+                    </div>
+                </div>
+                
+                <div class="code-snippet-display">
+                    <code>
+                        <iframe
```

Comment:
Try using an alternative to Cabon so that you code snippets become responsive, you can try Prism or Rainbow.

Reply:
Yeah in the next version i'll update that.